src/main/java/frc/robot/Constants2023.java

```diff
@@ -61,11 +61,13 @@ public class Constants2023 extends GameConstants {
     public static final double STEER_PID_FF_S = 0;//0.2;
     public static final double STEER_PID_FF_V = 0;//0.8;
     public static final double WHEEL_RADIUS = 0.0508;
+    public static final double ROBOT_RADIUS =  0.4191;
+
 
     public static final double MAX_VELOCITY = 4.0; // 4 meters per second
     public static final double MAX_ANGULAR_SPEED = Math.PI * 6; // 1 rotation per second
-    public static final double ROBOT_WIDTH = 0.5969;
-    public static final double ROBOT_LENGTH = 0.5969;
+    public static final double ROBOT_WIDTH = 0.91440;
```

Comment:
These new numbers don't seem correct. They should represent the width and length between swerve modules.

Reply:
I thought they were for the total robot width and length. I am going to make a new constant for those values

Reply:
nevermind I thought they were used by path planner, but the values are in settings.json, so they dont use the constants